portfolio/src/main/webapp/comments.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Sofia's Comments Tab</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+    </head>
+  <body>
+      <div id="my-content">
+          <h1>COMMENTS HERE</h1>
+          <p>See comments:</p>
+          <button onclick="getMessage()">Fetch hard-coded messages!</button>
+          <ul id="message-container"></ul>
+      </div>
+  </body>
+</html>
```

Comment:
Add a trailing newline please.

Reply:
Added! Can I ask if this is a Google style thing? I'm just very curious :)